app/src/reports/ReportProvider.tsx

```diff
@@ -30,57 +30,61 @@ function ReportProvider(props: { madLib: MadLib; setMadLib: Function }) {
   }
   switch (props.madLib.id as MadLibId) {
     case "disparity":
+      const dropdownOption = getPhraseValue(props.madLib, 1);
       return (
         <VariableDisparityReport
-          dropdownVarId={getPhraseValue(props.madLib, 1) as DropdownVarId}
+          key={dropdownOption}
```

Comment:
Maybe add a comment at the top of this whole switch statement as to why we use keys throughout